gen/denbi_portal_compute_center

```diff
@@ -99,7 +99,7 @@ my @users = values %$userStruc;
 # PRINT USERS TO JSON
 open FILE_USERS,">$fileUsers" or die "Cannot open $fileUsers: $! \n";
 binmode(FILE_USERS);
-print FILE_USERS JSON::XS->new->utf8->pretty->encode(\@users);
+print FILE_USERS JSON::XS->new->utf8->pretty->canonical->encode(\@users);
```

Comment:
Resulting `users.scim` file is missing `groupMembershipExpiration` property of JSON objects for new script version while old version has it. I'm not sure where is the problem yet.

Reply:
Original member data contained user, member, member-resource, user-facility and member-group attributes, added all.